src/App.test.js

```diff
@@ -1,82 +1,137 @@
 import { fireEvent, render, screen, waitFor } from '@testing-library/react';
-import '@testing-library/jest-dom'
-import { getAllMovieData } from './apiCalls.js'
-import { getSingleMovieData } from './apiCalls.js'
+import '@testing-library/jest-dom';
+import { getAllMovieData } from './apiCalls.js';
+import { getSingleMovieData } from './apiCalls.js';
 import App from './App';
-jest.mock('./apiCalls.js')
+import { MemoryRouter, Route } from 'react-router-dom';
+jest.mock('./apiCalls.js');
 
 describe('App', () => {
-  
   beforeEach(() => {
-    getAllMovieData.mockResolvedValueOnce({ 'movies': [{
-      "id": 694919,
-      "poster_path": "https://image.tmdb.org/t/p/original//6CoRTJTmijhBLJTUNoVSUNxZMEI.jpg",
-      "backdrop_path": "https://image.tmdb.org/t/p/original//pq0JSpwyT2URytdFG0euztQPAyR.jpg",
-      "title": "Money Plane",
-      "average_rating": 6.666666666666667,
-      "release_date": "2020-09-29"
+    getAllMovieData.mockResolvedValueOnce({
```

Comment:
Good change, having two different movies ensures testing is working with multiple sources/relates better to Movies.js layout